### PR TITLE
Raffles page

### PR DESCRIPTION
**Pull request by daniolivet**

### CODE REVIEW

File: src/components/Draw.astro [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+
+interface Props {
+  draw: any;
+  reverse: boolean
+}
+
+const { draw, reverse } = Astro.props;
+
+let rowClass = "flex-col sm:flex sm:flex-row gap-8 sm:gap-4 py-10";
```
Comment (davidrojom) [inline src/components/Draw.astro]: use class:list instead. Eg

     <div class:list={{
        "class1 class2 class3": reverse,
      }}>....</div>

File: src/components/Draw.astro
Comment (davidrojom) [inline src/components/Draw.astro:1]: rename it to raffle instead

File: src/components/Navigation.astro [hunk 2]
```diff
@@ -53,6 +53,13 @@ import If from "./utils/If.astro";
           <a href="/#speakers" class="px-3"> Speakers</a>
         </li>
       </If>
+      <If condition={!!DATA.draws.length}>
```
Comment (davidrojom) [inline src/components/Navigation.astro]: rename to raffles

File: src/components/Navigation.astro [hunk 3]
```diff
@@ -53,6 +53,13 @@ import If from "./utils/If.astro";
...
+        <li
+          class="cursor-pointer grid place-content-center hover:bg-gray-100 dark:hover:bg-gray-700 rounded-lg"
+        >
+          <a href="/draws" class="px-3"> Sorteos</a>
```
Comment (davidrojom) [inline src/components/Navigation.astro]: rename

File: src/pages/draws/index.astro [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+import Draw from "../../components/Draw.astro";
+import AnimatedOnViewportEnter from "../../components/utils/AnimatedOnViewportEnter.astro";
+import { DATA } from "../../data/data";
+import Layout from "../../layouts/Layout.astro";
+
+---
+
+<Layout title={"Sorteos"} description={""} lang="es">
```
Comment (davidrojom) [inline src/pages/draws/index.astro]: Add description, remove braces

**Review by davidrojom — CHANGES_REQUESTED** [review]

File: src/components/Draw.astro
Comment (daniolivet) [reply src/components/Draw.astro:1]: donete

File: src/components/Raffle.astro [hunk 5]
```diff
@@ -1,24 +1,27 @@
...
 
-const { draw, reverse } = Astro.props;
+const { raffle, reverse } = Astro.props;
 
 let rowClass = "flex-col sm:flex sm:flex-row gap-8 sm:gap-4 py-10";
 if( reverse ) {
     rowClass = "bg-yellow-100 flex flex-col flex-col-reverse sm:flex-row sm:flex-row-reverse gap-8 sm:gap-4 py-10"
 }
-
+console.log(reverse);
```
Comment (davidrojom) [inline src/components/Raffle.astro]: Remove console.log

**Review by davidrojom — CHANGES_REQUESTED** [review]

**Pull request merged**